src/main/kotlin/no/nav/syfo/services/UpdateInfotrygdService.kt

```diff
@@ -382,4 +387,73 @@ suspend fun sendInfotrygdOppdateringAndValidationResult(
             typeSMinfo?.periode?.arbufoerFOM ?: throw RuntimeException("Unable to find første fraværsdag in IT")
         }
     }
+
+    fun produceManualTaskAndSendValidationResults(
+        kafkaProducer: KafkaProducer<String, ProduceTask>,
+        receivedSykmelding: ReceivedSykmelding,
+        validationResult: ValidationResult,
+        navKontorNr: String,
+        loggingMeta: LoggingMeta,
+        oppgaveTopic: String,
+        sm2013BehandlingsUtfallToipic: String,
+        kafkaproducervalidationResult: KafkaProducer<String, ValidationResult>,
+        itfh: InfotrygdForespAndHealthInformation,
+        helsepersonellKategoriVerdi: String,
+        jedis: Jedis
+    ) {
+        sendRuleCheckValidationResult(receivedSykmelding, kafkaproducervalidationResult,
+                validationResult, sm2013BehandlingsUtfallToipic, loggingMeta)
+        try {
+            val perioder = itfh.healthInformation.aktivitet.periode.sortedBy { it.periodeFOMDato }
+            val forsteFravaersDag = finnForsteFravaersDag(itfh, perioder.first(), loggingMeta)
+            val sha256String = sha256hashstring(createInfotrygdBlokk(
+                    itfh, perioder.first(), receivedSykmelding.personNrPasient, LocalDate.of(2019, 1, 1),
```

Comment:
Hva er 1/1 2019, og 0?

Reply:
1/1  2019 er signaturdato, denne vil endres ved ver sykmelding, og detfor bør vi ta denne ut av sammeligins verdien for duplikatsjekkt.
0 er legens tssid, har rettet dette nå, pga vi kan ende opp med ein null